### ISSUE TITLE
Can I open a multiples pokebuddys?

### ISSUE THREAD

**Issue by edwarbad23** [finndev/PokeBuddy#1170]
I need botting a multiples account so this is my question!! Thnx

**Comment by Duero421** [comment]
No

**Comment by NegroSorete** [comment]
Yes you can... but you probability get banned

**Comment by Rynier** [comment]
Yes you can. I run 2 bots at the same time, just make sure they run in the same location if you worried about ip ban.
